web-ng/src/app/components/observation-form/observation-form.component.css

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.observation {
+    margin: 16px;
+}
+.observation .field-label {
+    font-size: 12px;
+    line-height: 16px;
+    color: #70757A;
+    margin-bottom: 3px;
+}
+.observation .field-response {
+    font-size: 14px;
+    line-height: 20px;
+    color: #3C4043;
```

Comment:
Please use `mat-color()` instead.

Reply:
i deleted custom color. it will be default now

Reply:
Thank you!